code/src/java/pcgen/util/PJEP.java

```diff
@@ -301,7 +301,7 @@ else if (parent instanceof PlayerCharacter)
 	 * @return Returns the variableSource.
 	 */
 	@Deprecated
-	protected String getVariableSource()
+	private String getVariableSource()
```

Comment:
@grimreaper I guess you should change this back to protected then and if Tom doesn't have any more objections I'll merge it.